.github/workflows/manual.yml

```diff
@@ -1,74 +1,81 @@
-# This is a basic workflow that is manually triggered
+name: Run Mass Dump
 
-name: Run Packet Dumper
-
-# Controls when the action will run. Workflow runs when manually triggered using the UI
-# or API.
 on:
   workflow_dispatch:
-    # Inputs the workflow accepts.
-    inputs:
-      version:
-        # Friendly description to be shown in the UI instead of 'name'
-        description: 'Version to dump'
-        # Default value if no value is explicitly provided
-        default: '1.16'
-        # Input has to be provided for the workflow to run
-        required: true
-
-# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+  
 jobs:
-  # This workflow contains a single job called "greet"
-  dump:
-    # The type of runner that the job will run on
+  dump-packets:
+
     runs-on: ubuntu-latest
+
     strategy:
       matrix:
-        node-version: [14.x]
+        mcVersionIndex: ["1.8", "1.9", "1.10", "1.11", "1.12", "1.13", "1.14", "1.15", "1.16"]
 
-    # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
+    
+    - name: Use Node.js 14.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 14.x
+        
     - name: Setup Java JDK
-      uses: actions/setup-java@v1.4.3
+      uses: actions/setup-java@v1
       with:
         java-version: 1.8
         java-package: jre
+
+    - name: Use Dumper
+      run: |
+        npm i -g prismarine-packet-dumper
+        mkdir ${{ matrix.mcVersionIndex }}
+        dumpPackets -v ${{ matrix.mcVersionIndex }} -o ${{ matrix.mcVersionIndex }}
+        mkdir ${{ matrix.mcVersionIndex }}/metadata
```

Comment:
Why are you recreating the metadata folder here ?
Either don't do it or create it in dumper

Reply:
by default packet dumper just theows the json file in the root of the dir, i can make packet dumper go to a folder ig

Reply:
done